walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -166,16 +166,16 @@ Modify the Fuse Aggregation App to aggregate flights data from the Arrivals & De
 .. Select the *Profile* menu, then *Preferences*.
 .. Choose the *Commiter* option under the *Git* heading.
 .. Set a *Name* and *Email*.
-.. Select *Save* then *Close*.
+.. Click *Save* then *Close*.
```

Comment:
Suggestion: "Save → Close".

It might not be fitting in the grand scheme of things but it's something that was used in https://docs.aerogear.org/aerogear/latest/mobile-metrics.html

Reply:
Yep, will change it.  It would be consistent with the way its done in other parts of the doc as well.